include/api.php

```diff
@@ -1754,7 +1755,7 @@ function api_statuses_mentions($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.